tests/pivots/pivot_menu_items.test.ts

```diff
@@ -368,7 +368,7 @@ describe("Pivot fix formula menu item", () => {
   test("Fixing the formula take into account the arguments of PIVOT()", () => {
     // prettier-ignore
     const grid = {
-     A1: "Customer", B1: "Price", C1: "Date",     E1: "=PIVOT(1, 1, false, false, 1)",
+     A1: "Customer", B1: "Price", C1: "Date",     E1: "=PIVOT(1, 1, false, false, 1, false)",
```

Comment:
Why this change ? You change the result of the pivot function in the test (now the measure isn't shown), but the test didn't have to change ?

Reply:
Nvm I understood, before hiding the headers also hid the measures. The change might break some spreadsheets then ? Are we fine with that ?

Reply:
Yes it might break some spreadsheets, and from what we discussed yesterday with @LucasLefevre it was fine.
Another solution is to make this new attribute false by default and did not take it into account if `include_columns_title` is true to keep the current behavior, but meh...

As soon as it's not a traceback, I think I will be able to sleep tonight